recipes-overlayed/kselftests/kselftests_4.12-rc7.bb

```diff
@@ -124,20 +130,24 @@ FILES_${PN}-dbg += "${bindir}/kselftests/*/.debug"
 # make[1]: *** [test_verifier] Error 1
 ALLOW_EMPTY_${PN}-bpf = "1"
 
-# FIXME net target builds most of the binaries, but reuseport_bpf_numa depends on libnuma,
-# which is not availbale on ARM, failing entire test case
-ALLOW_EMPTY_${PN}-net = "1"
+# FIXME breakpoints currently fails because of missing
+# symbols from glibc. C.f. this bug:
+#   https://sourceware.org/bugzilla/show_bug.cgi?id=21286
+ALLOW_EMPTY_${PN}-breakpoints = "1"
```

Comment:
It first tries to build breakpoint_test, fails, and exits immediately -- doesn't get a chance to build step_after_suspend_test.